worker.js

```diff
@@ -115,12 +148,10 @@ app.post('/token', async (c) => {
 				// i had issues doing this any other way?
 				const memberResp = await memberPromise
 				const memberJson = await memberResp.json()
-
+				console.log(memberJson)
```

Comment:
Debugging is great, but if I have logpush enabled on the Worker I don't want to have user information spammed over my logs unless there's an error.  Same for other instances of this.

Reply:
Ik, gonna revert that later.
Putting this pr again in draft